app/vip-config/__autotest-checker.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Inpsyde\Vip;
+
+// phpcs:disable Inpsyde.CodeQuality.Psr4
+
+class AutotestChecker
```

Comment:
Should we add some documentation for those (probably in class file header should be okay) about this? 👀

Reply:
Already on it ;)

Reply:
pushed